cfme/tests/candu/test_vm_graph.py

```diff
@@ -2,21 +2,32 @@
 
 from cfme import test_requirements
 from cfme.common.vm import VM
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider
 from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.wait import wait_for
 
 
 pytestmark = [
     pytest.mark.tier(3),
     test_requirements.c_and_u,
     pytest.mark.usefixtures('setup_provider'),
-    pytest.mark.provider([VMwareProvider],
+    pytest.mark.provider([VMwareProvider, RHEVMProvider, EC2Provider, AzureProvider],
                          required_fields=[(['cap_and_util', 'capandu_vm'], 'cu-24x7')])
 ]
 
 VM_GRAPHS = ['vm_cpu', 'vm_cpu_state', 'vm_memory', 'vm_disk', 'vm_network']
 
 
+# ToDo: Currently disk activity for EC2 not collecting due to infra issue.
```

Comment:
**QUESTION:** Is there some sort of upstream bug/issue we could reference in this comment?

Reply:
No, we are working on it. Its volume issue basically. 
https://forums.aws.amazon.com/thread.jspa?threadID=108613